mods/default/nodes.lua

```diff
@@ -1990,12 +1990,19 @@ function default.register_chest(name, d)
 	local def_closed = table.copy(def)
 
 	def_opened.mesh = "chest_open.obj"
+	for i = 1, #def_opened.tiles do
+		if type(def_opened.tiles[i]) == "string" then
+			def_opened.tiles[i] = {name = def_opened.tiles[i], backface_culling = true}
+		elseif def_opened.tiles[i].backface_culling == nil then
```

Comment:
`elseif def_opened.tiles[i].backface_culling != true then`
To override if it i set to 'false'?